packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -155,15 +157,62 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
         // TODO: Mark msg.sender as already voted on electionVotes
     }
 
-    function _isNextEpochNomination() internal view virtual returns (bool) {
-        return false;
+    function epochFinised() external view override returns (bool) {
+        if (_electionStore().epochStart == 0) {
```

Comment:
I'd avoid reverting in view functions as much as possible.  (Also applies in other parts of the file)